tests/tests/swfs/avm2/edittext_autosize_align/test.toml

```diff
@@ -0,0 +1,7 @@
+num_ticks = 1
+
+[image_comparisons.output]
+tolerance = 128
```

Comment:
To be sure, why such a high tolerance?

Reply:
It might seem that the tolerance is pretty high, but for tests which check the placement of black rectangles over white background, it basically means a half-pixel tolerance. It's pretty common that we're off by half a pixel, e.g. in cases of border corners (different line rasterization algorithms), or glyph placement (font hinting probably?).

Reply:
But is it _everywhere_ that's wrong, or just some places? Can we reduce the tolerance and increase the outliers? I'm a little worried by "every pixel can be half wrong" 😅

Reply:
I could alternatively do low tolerance + max_outliers, but TBH after making so many text tests I got bored of counting these outliers and making sure they are at the right places.

Reply:
The solution would be to provide max outliers with max tolerance for each one, so that we could say "there are max 144 outliers, but each one shouldn't be more off than 128".